manifests/action_install_aem_profile.pp

```diff
@@ -29,6 +29,7 @@
 
   aem_curator::install_aem_profile { "${aem_id}: Install AEM profile ${aem_profile}":
     aem_artifacts_base      => $aem_artifacts_base,
+    aem_license_base        => $aem_license_base,
```

Comment:
Please update changelog with this new param since it is a mandatory param and user will be impacted.

Reply:
Done